addons/l10n_nl/data/account_account_tag.xml

```diff
@@ -136,5 +136,67 @@
             <field name="applicability">accounts</field>
         </record>
 
+        <record id="account_tag_immat" model="account.account.tag">
+            <field name="name">Intangible Fixed Asset</field>
+            <field name="applicability">accounts</field>
```

Comment:
I think we should also add the country id here, to restrict the tag to the localisation